### PR TITLE
implement task 5

### PR DESCRIPTION
**Pull request by zhenokin**
1) FE PR: https://github.com/zhenokin/nodejs-aws-fe/pull/3
2) https://dlnhu3eqgkqh.cloudfront.net/
3) Additional scope:
- [x] async/await is used in lambda functions
- [ ] importProductsFile lambda is covered by unit tests
- [x]  At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder

7/8

### CODE REVIEW

File: import-service/handlers/importFileParser.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import * as AWS from 'aws-sdk';
+import { BUCKET, REGION } from '../constants/constants';
+import csv from 'csv-parser';
+
+export const importFileParser = (event, _context) => {
```
Comment (valakar) [inline import-service/handlers/importFileParser.ts:5]: I couldn't check this, but can you try to load file with some data and check if all of them properly logged in console of your fileParse lambda?  (and also check if uploaded filed is properly deleted)
It looks like you are not waiting for stream to finish. 

This criteria may not be met:

`+1 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)`

Comment (zhenokin) [reply import-service/handlers/importFileParser.ts:5]: > I couldn't check this, but can you try to load file with some data and check if all of them properly logged in console of your fileParse lambda? (and also check if uploaded filed is properly deleted)
> It looks like you are not waiting for stream to finish.
> 
> This criteria may not be met:
> 
> `+1 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)`

Hello
I checked this case and it works fine. The **importFileParser** is not **async** function (there is no **async** operator), so a lambda works until the event loop will be empty

**Comment by viacheslav-karnaukh** [comment]
Looks good. Everything is done. Some additional work is also done (except unit tests for `importProductsFile`). I couldn't find your CORS config for S3 bucket. But frontend app works correctly. So,have you applied CORS config manually?

**Review by viacheslav-karnaukh — APPROVED** [review]

**Comment by zhenokin** [comment]
> Looks good. Everything is done. Some additional work is also done (except unit tests for `importProductsFile`). I couldn't find your CORS config for S3 bucket. But frontend app works correctly. So,have you applied CORS config manually?

yes
image

**Pull request merged**